Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportEntitiesDataMapping.cs

```diff
@@ -424,15 +424,29 @@ private static void SetRunnerChildCounts(GingerExecutionEngine runner, AccountRe
 
             foreach (BusinessFlow businessFlow in runner.BusinessFlows)
             {
-                ChildExecutableItemsCountActivity = ChildExecutableItemsCountActivity + businessFlow.Activities.Count(x => x.Active && (x.Status == eRunStatus.Passed || x.Status == eRunStatus.FailIgnored || x.Status == eRunStatus.Failed || x.Status == eRunStatus.Blocked));
+                if(IsStart)
+                {
+                    ChildExecutableItemsCountActivity = ChildExecutableItemsCountActivity + businessFlow.Activities.Count(x=> x.Active);
```

Comment:
are we now collecting "Executed" count on Start for calculating the progress during execution?
how we did it before?

Reply:
for calculation of count at start & end we were using same function which is bug.